2025-04-30 16:57:05,292 - DEBUG    - Invalid IP bulb array of length: 909 - ( test_db_validations.py : 239 )
2025-04-30 16:57:05,741 - DEBUG    - Bulb creations with invalid IPs failed as expected - ( test_db_validations.py : 247 )
